### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Secrets-Emperor**
### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

**Comment by Secrets-Emperor** [comment]
I Humble request to accept my Changing

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Please see the [Changelog](CHANGELOG.md) for more details about a given release.
 
 ## Konnect
 Kong Inc. offers commercial subscriptions that enhance the Kong API Gateway in a variety of ways. Customers of Kong's [Konnect](https://konghq.com/kong-konnect/) subscription take advantage of additional gateway functionality, commercial support, and access to Kong's managed (SaaS) control plane platform. The Konnect platform features include real-time analytics, a service catalog, developer portals, and so much more! [Get started](https://konghq.com/get-started/) with Konnect.
-
+Say Hello to Kongs
```
Comment (juzhiyuan) [inline README.md:88]: May I know why we need this change? 😄

**Comment by Secrets-Emperor** [comment]
Because it looks more friendly to other contributors.

File: README.md (hunk 1 above)
Comment (Secrets-Emperor) [reply README.md:88]: Because it looks friendly to new contributors like me 😊

**Comment by mayocream** [comment]
We have [contributing section](https://github.com/Kong/kong#contributing) welcoming new contributors.

**Comment by mayocream** [comment]
Feel free to re-open this and explain why we need this PR.
You can also create a discussion [here](https://github.com/Kong/kong/discussions) sharing your ideas about improving docs to new contributors :)

**Comment by Secrets-Emperor** [comment]
I am new in this but I really understand what you want to say and I really appreciate it thanks for your guidance 😊

**Comment by Secrets-Emperor** [comment]
And if you don't mind can I know why my pr is rejected when I give a sufficient example

**Pull request closed without merging**